Project-Euler/Project-Euler/Program.cs

```diff
@@ -11,28 +11,93 @@ class Program
         static void Main()
         {
             PrintIntro();
-            PrintMenuOptions();//Eventually this method will not be called in main directly. This is for testing purposes.
+
+            while (true)
+            {
+                MenuOptions userSelection = GetMenuSelection();
+
+                if (userSelection == MenuOptions.A)
+                {
+                    DynamicProblemSelector.PrintAllProblems();
+                }
+                else if (userSelection == MenuOptions.S)
+                {
+                    DynamicProblemSelector.PrintProblem(GetProblemNumber(), typeof(Problems));
+                }
+                else // User Chose to Quit
+                {
+                    return;
+                }
+            }
         }
 
+        public enum MenuOptions { A, S, Q };
+
         public static void PrintIntro()
         {
             Console.WriteLine("\nWelcome To My Project Euler Repository!");
             //Add a couple more lines, maybe?
         }
 
-        public static void PrintMenuOptions() //This method will be private once I write the method that will call it. blah blah
+        public static MenuOptions GetMenuSelection()
+        {
+            while (true)
+            {
+                PrintMenuOptions();
+
+                try
+                {
```

Comment:
Why did you put this in a try block? 

1. I don't think this is necessary
2. It's very bad practice to use a try with an empty catch, you are basically just ignoring errors

Google about empty catch blocks and why they are bad and also when is the best time to use a try-catch